### PR TITLE
Refactor pull_request_package

### PR DESCRIPTION
**Pull request by vpereira**
Rewrite of the pull_request_package. 

Things done in this PR:

- move code to an external gem
- make it possible to configure parameters
- dockerize it

### CODE REVIEW

File: pull_request_package/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM opensuse/tumbleweed
+
+RUN zypper  --non-interactive --quiet ref \
+ && zypper -n in osc ruby2.6 ruby2.6-devel git gcc make autoconf zlib-devel libxml2-devel libxslt-devel
+RUN gem install bundler
+RUN bundle.ruby2.6 config build.nokogiri --use-system-libraries
+COPY .  /home/puller/pull_request_package
+RUN cd /home/puller/pull_request_package && bundler.ruby2.6 install
```
Comment (hennevogel) [inline pull_request_package/Dockerfile]: Please move up the `WORKDIR` again.

File: pull_request_package/Gemfile [hunk 2]
```diff
@@ -4,6 +4,5 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
-gem 'octokit'
-gem 'nokogiri'
-gem 'activemodel'
+gem 'slop'
+gem 'pull_request_builder', github: 'vpereira/pull_request_builder'
```
Comment (hennevogel) [inline pull_request_package/Gemfile]: Hm shouldn't the gem's source be here also?

Comment (vpereira) [reply pull_request_package/Gemfile]: I wasn't thinking on pack this as gem, just the ```pull_request_builder``` but if you want, I can do it

Comment (hennevogel) [reply pull_request_package/Gemfile]: Why not?

Comment (coolo) [reply pull_request_package/Gemfile]: I don't think this way you can build the container on OBS

**Pull request closed without merging**